test/unit/views/payment-sheet-views/apple-pay-view.js

```diff
@@ -338,12 +395,35 @@ describe('ApplePayView', () => {
                 done();
               };
 
-              testContext.buttonClickHandler();
+              button.onclick();
               testContext.fakeApplePaySession.onpaymentauthorized({
                 payment: { token: 'foo' }
               });
             }
           );
+
+          test(
+            'can start a new session',
```

Comment:
```suggestion
            'can start a new session after a tokenization error',
```
Not sure if this makes sense but noticed the other ones have a similar "when" clause.